test/utils/image/manifest.go

```diff
@@ -245,6 +247,7 @@ func initImageConfigs() map[int]Config {
 	configs[VolumeGlusterServer] = Config{e2eRegistry, "volume/gluster", "1.0"}
 	configs[VolumeRBDServer] = Config{e2eRegistry, "volume/rbd", "1.0.1"}
 	configs[WindowsNanoServer] = Config{e2eGcRegistry, "windows-nanoserver", "v1"}
+	configs[HugePageTester] = Config{"docker.io", "odinuge/hugepage-tester", "1.0"}
```

Comment:
I'm not positive, but my strong guess is that the project will be hesitant to begin relying upon images hosed on a contributors "personal" docker.io account. I wonder if there's a way (if you would be willing to) transfer ownership to the main e2e registry...

Although I'm not sure how that process works. @tallclair - looks like you added an image recently? do you have context?

Reply:
Yeah! in the original PR i commented that this had to be sorted out before being merged (tho i had to do this in order to test). :smile: I guess it belongs inside `e2eRegistry`. Building of the cross-arch image should be good to go using the `test/images/image-util.sh` :smile:

Reply:
Cool! Is there the possibility to host this before this pr is merged? If not, we can most likely run the test suite with it hosted on `docker.io`, although I would want someone else from the project to confirm that :)

Reply:
I definitely think that the best solution is to build and upload this `e2eRegistry` _before_ getting it merged. I have no idea about who can do it. 😅 

Guess we can sort out all the other things first tho, and look at this when we think it is ready do get merged.

Reply:
cc @dims it seems like you may know about this process based on your recent email around a staging repo for images.